src/services/database_service.py

```
@@ -32,7 +34,9 @@ class DatabaseService:
     and source information tracking with comprehensive error handling and retries.
     """
 
-    def __init__(self, supabase_client: SupabaseClientProtocol | None = None, embedding_service: EmbeddingService | None = None):
+    def __init__(
+        self, supabase_client: SupabaseClientProtocol | None = None, embedding_service: EmbeddingService | None = None
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)